### PR TITLE
pester : Change the installation steps

### PR DESCRIPTION
**Pull request by alexchapar**
## Description
Change the installation steps since they were based on an older version ( < 5 ).

## Related Tickets & Documents
No ticket

### Check Hashes?

- [x] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### Check Url?

- [x] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### Check the last version?

- [x] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### Added a description in the manifest?

- [x] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### Beautified the json manifest format?

- [x] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added some notes in the manifest?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added/Needed any dependencies?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed

## Added/Needed any pre-installation steps to perform?

- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

## Added/Needed any post-installation steps to perform?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed

## [optional] What gif/emoji best describes this PR or how it makes you feel?
:unamused:

### CODE REVIEW

**Comment by BotServiceAutomation** [comment]
All changes look good.

Wait for review from human collaborators.
### pester

- [x] Description
- [x] License
- [x] Hashes
- [x] Checkver
- [x] Autoupdate

**Review by Bibz87-CG — APPROVED** [review]

**Pull request merged**
